common.mk

```diff
@@ -17,25 +17,24 @@ endif
 script_env = \
 	SKIP_SQUASH=$(SKIP_SQUASH)                      \
 	UPDATE_BASE=$(UPDATE_BASE)                      \
-	VERSIONS="$(VERSIONS)"                          \
 	OS=$(OS)                                        \
-	VERSION="$(VERSION)"                            \
 	BASE_IMAGE_NAME=$(BASE_IMAGE_NAME)              \
 	OPENSHIFT_NAMESPACES="$(OPENSHIFT_NAMESPACES)"
 
 .PHONY: build
-build: manpages
-	$(script_env) $(build)
+build: $(VERSIONS)
```

Comment:
This will cause parallel build of multiple image versions, right?  What's the status of docker bug discussed in  sclorg/postgresql-container/pull/149 ?